### PR TITLE
Icon request

### PR DESCRIPTION
**Pull request by aloxe**
providing icons for my own request: creditas curve ebtart freedcamviewer mapycz okcagent pidlitacka rbkey stelarium tricount zonky

### CODE REVIEW

File: app/src/main/res/xml/appfilter.xml [hunk 1]
```diff
@@ -4005,9 +4038,13 @@
     <!-- zmNinja -->
     <item component="ComponentInfo{com.pliablepixels.zmninja_pro/com.pliablepixels.zmninja_pro.MainActivity}" drawable="zmninja" />
 
+    <!-- Zonky -->
+    <item component="ComponentInfo{cz.zonky/cz.zonky.app.AppActivity}" drawable="zonky"/>
+
     <!-- Zoo for Zotero -->
     <item component="ComponentInfo{com.mickstarify.zooforzotero/com.mickstarify.zooforzotero.MainActivity}" drawable="zooforzotero"/>
 
+
```
Comment (dkanada) [inline app/src/main/res/xml/appfilter.xml:4047]:
```suggestion
```

File: app/src/main/res/xml/appfilter.xml [hunk 2]
```diff
@@ -3701,11 +3731,14 @@
...
 
+    <!-- Tricount -->
+    <item component="ComponentInfo{com.tribab.tricount.android/com.tribab.tricount.android.view.activity.SplashActivity}" drawable="tricount"/>
+
     <!-- Trigger -->
     <item component="ComponentInfo{com.example.trigger/com.example.trigger.MainActivity}" drawable="trigger" />
 
     <!-- Tumblr -->
-    <item component="ComponentInfo{com.tumblr/com.tumblr.ui.activity.JumpoffActivity}" drawable="tumblr" />
+    <item component="ComponentInfo{com.tumblr/com.ui.activity.JumpoffActivity}" drawable="tumblr" />
```
Comment (dkanada) [inline app/src/main/res/xml/appfilter.xml:3741]:
```suggestion
    <item component="ComponentInfo{com.tumblr/com.tumblr.ui.activity.JumpoffActivity}" drawable="tumblr" />
    <item component="ComponentInfo{com.tumblr/com.ui.activity.JumpoffActivity}" drawable="tumblr" />
```

File: app/src/main/res/xml/appfilter.xml [hunk 3]
```diff
@@ -1742,7 +1756,7 @@
     <item component="ComponentInfo{com.lyndir.masterpassword/com.lyndir.masterpassword.EmergencyActivity}" drawable="masterpassword" />
 
     <!-- Material Islands -->
-    <item component="ComponentInfo{com.hippotech.materialislands/com.hippotech.materialislands.MainActivity_}" drawable="materialislands" />
+    <item component="{com.hippotech.materialislands/com.hippotech.materialislands.MainActivity_}" drawable="materialislands" />
```
Comment (dkanada) [inline app/src/main/res/xml/appfilter.xml:1759]:
```suggestion
    <item component="ComponentInfo{com.hippotech.materialislands/com.hippotech.materialislands.MainActivity_}" drawable="materialislands" />
```

**Comment by aloxe** [comment]
Thanks for that fix, is it ready for merge?

**Comment by dkanada** [comment]
Yeah, it will be in the next update :)

**Pull request merged**